services/common/src/components/forms/FormWrapper.tsx

```diff
@@ -8,6 +8,7 @@ export interface IFormContext {
   isEditMode: boolean;
   isModal: boolean;
   formName: string;
+  onReset: () => void | Promise<void>;
```

Comment:
There were a couple forms with an onReset so I added this in!
Basically, new antd _does not have_ an onReset parameter. 
So, to actually use this, I also added `RenderResetButton`. It resets the form and calls this function (if it exists). It's more for resetting search forms than for cancelling edits, so it doesn't prompt you with "are you sure?"